src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,53 @@
+.clock {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
```

Comment:
You can remove space

Reply:
I don't see a problem here actually, I've just made it for better readability of code and it doesn't do any harm.

Reply:
> You can remove space

I don't see a problem here actually, I've just made it for better readability of code and it doesn't do any harm.